frontend/packages/client/src/components/ProposalInformation/Information.js

```diff
@@ -84,28 +85,38 @@ export default function Information({
         <InfoBlock
           title={'Token required'}
           component={
-            <a
-              href={'https://flowscan.org/'}
-              rel="noopener noreferrer"
-              target="_blank"
-              className="is-underlined has-text-grey p-0 small-text"
-              style={{ height: '2rem !important' }}
-            >
-              <span className="mr-2">{`$${tokenName?.toUpperCase()}`}</span>
-              <Svg name="LinkOut" width="12" height="12" />
-            </a>
+            contractAddr ? (
+              <a
+                href={`https://flowscan.org/account/${contractAddr}`}
+                rel="noopener noreferrer"
+                target="_blank"
+                className="is-underlined has-text-grey p-0 small-text"
+                style={{ height: '2rem !important' }}
+              >
+                <span className="mr-2">{`$${tokenName?.toUpperCase()}`}</span>
+                <Svg name="LinkOut" width="12" height="12" />
+              </a>
+            ) : (
+              <div
+                className="has-text-grey p-0 small-text"
+                style={{ height: '2rem !important' }}
+              >
+                <span className="mr-2">{`$${tokenName?.toUpperCase()}`}</span>
+                <Svg name="LinkOut" width="12" height="12" />
+              </div>
+            )
```

Comment:
just updated href={'https://flowscan.org/'} to href={`https://flowscan.org/account/${contractAddr}`} and if no addr is passed then no link is shown